packages/web/src/pages/sign-up-page/pages/SelectGenrePage.tsx

```diff
@@ -52,40 +53,76 @@ export const SelectGenrePage = () => {
     [dispatch, navigate]
   )
 
+  const { isMobile } = useMedia()
+
   return (
-    <div>
-      <div>
-        <p>{displayName}</p>
-        <p>{handle}</p>
-      </div>
-      <h1>{messages.header}</h1>
-      <p>{messages.description}</p>
+    <Flex direction='column' gap={isMobile ? '2xl' : '4xl'}>
+      <AccountHeader />
+      <Flex direction='column' gap={isMobile ? 'xl' : '2xl'}>
+        <Flex direction='column' gap={isMobile ? 's' : 'l'} ph='l'>
+          <Text
+            variant='heading'
+            size={isMobile ? 'm' : 'l'}
+            color='heading'
+            css={{ textAlign: isMobile ? 'left' : 'center' }}
+          >
+            {messages.header}
+          </Text>
+          <Text
+            variant='body'
+            size={isMobile ? 'm' : 'l'}
+            css={{ textAlign: isMobile ? 'left' : 'center' }}
+          >
+            {messages.description}
+          </Text>
+        </Flex>
 
-      <Formik initialValues={initialValues} onSubmit={handleSubmit}>
-        {({ values, setValues }) => {
-          const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
-            setValues({ ...values, [e.target.name]: !values[e.target.name] })
-          }
+        <Formik initialValues={initialValues} onSubmit={handleSubmit}>
+          {({ values, setValues }) => {
+            const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+              setValues({ ...values, [e.target.name]: !values[e.target.name] })
+            }
 
-          return (
-            <Form>
-              {genres.map((genre) => {
-                const { label, value } = genre
-                return (
-                  <GenrePill
-                    key={value}
-                    label={label}
-                    name={value}
-                    checked={!!values[value]}
-                    onChange={handleChange}
-                  />
-                )
-              })}
-              <Button type='submit'>{messages.continue}</Button>
-            </Form>
-          )
-        }}
-      </Formik>
-    </div>
+            return (
+              <Flex
+                as={Form}
```

Comment:
ooh nice. Just curious, would this pass the formik props to children properly?

```
<Flex as={Form}>
	{(formikProps) => <Stuff />}
</Flex>
```

Reply:
so form doest have the render-prop pattern, just formik, so could maybe flex as={Formik} ?

Reply:
yeah I mean we don't need the props here necessarily, was just curious if that would work

Reply:
Yeah I wonder if the as prop is smart enough to also change the children type